lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -1,9 +1,10 @@
-;(function (define) {
+;(function (define, interpolate) {
```

Comment:
I now understand your point better... if ``interpolate`` was brought in on demand, then it wouldn't be defined at this point. I'm pretty sure that it is defined in the environment and that ``gettext`` just provides a shim, but it does make it look confusing. I wish there were a cleaner way to bring it in too. Maybe we should provide it as a shim too.